gfx/portraits/portrait_modifiers/00_hairstyles_scripted_characters.txt

```diff
@@ -503,6 +503,10 @@ hairstyles_scripted_characters = {
                 add = 200
 				is_blackhand_trigger = yes
             }
+            modifier = {
+                add = 200
+				is_maim_trigger = yes
```

Comment:
Prob you would like to use a new hairstyle for Maim, `male_hair_indian_03_wc`. It's bald with a bun.

Reply:
Updated in newest commit